src/components/tables/EntityTable.tsx

```diff
@@ -43,14 +43,19 @@ import {
     TableStatuses,
 } from 'types';
 import { getEmptyTableHeader, getEmptyTableMessage } from 'utils/table-utils';
-import DateFilter from './Filters/Date';
 import { RowSelectorProps } from './RowActions/types';
 
+export interface ColumnProps {
+    field: string | null;
+    headerIntlKey?: string | null;
+    renderHeader?: (
+        index: number,
+        storeName: SelectTableStoreNames
+    ) => ReactNode;
+}
```

Comment:
Pulled out this type so we could import in the `Rows` files when setting up a column array.